test/import/test_git.rb

```diff
@@ -741,3 +748,114 @@ def import(**options)
         importer.import(pkg)
     end
 end
+
+describe Autobuild::Git do
```

Comment:
This block should not be repeated. Just add the new describe blocks to the existing toplevel `describe` block.